engine/src/cubos/engine/gizmos/renderer.cpp

```diff
@@ -101,20 +176,20 @@ void GizmosRenderer::initCutConePrimitive()
     cutConePrimitive.vaDesc.elements[0].buffer.index = 0;
     cutConePrimitive.vaDesc.elements[0].buffer.offset = 0;
     cutConePrimitive.vaDesc.elements[0].buffer.stride = 3 * sizeof(float);
-    cutConePrimitive.vaDesc.shaderPipeline = pipeline;
+    cutConePrimitive.vaDesc.shaderPipeline = drawPipeline;
 
     float verts[CutConeVertsPerBase * 6];
 
-    renderDevice->setShaderPipeline(pipeline);
+    renderDevice->setShaderPipeline(drawPipeline);
     cutConePrimitive.vb = renderDevice->createVertexBuffer(sizeof(verts), verts, Usage::Dynamic);
     cutConePrimitive.vaDesc.buffers[0] = cutConePrimitive.vb;
     cutConePrimitive.va = renderDevice->createVertexArray(cutConePrimitive.vaDesc);
 
-    unsigned int indices[12 * CutConeVertsPerBase - 12];
+    std::size_t indices[12 * CutConeVertsPerBase - 12];
```

Comment:
This is wrong, the buffer takes an `UInt`, not a `std::size_t`. This only works on your machine because you're compiling this for x86, which means they both have the same size.